app/javascript/elm/src/Main.elm

```diff
@@ -84,8 +85,9 @@ defaultModel =
     , isStreaming = True
     , selectedSession = NotAsked
     , logoNav = ""
-    , linkIcon = ""
-    , resetIcon = ""
+    , linkIcon = Path.fromString ""
+    , resetIcon = Path.fromString ""
+    , resetIconWhite = Path.fromString ""
```

Comment:
You could also have a method `Path.emptytIcon` in Path.elm

Reply:
I vote for `Path.empty`. I don't think `Path` should know about icons or not

Reply:
https://github.com/HabitatMap/AirCasting/commit/9b64befbe5d86e8261bf6bf0e4c16e4d7dd25a52